.github/workflows/Testear_versiones.yml

```diff
@@ -0,0 +1,30 @@
+name: Testear Versiones Python
+
+on:
+  push:
+    branches:
```

Comment:
Supongo que un poco innecesario, nunca pusheamos directamente a main, y se supone que tiene que servir para todos los objetivos, asi que "hardcodear" el 6 no creo sea buena idea.

Reply:
Eso ha sido simplemente para probar que funcionaba, ya que los push que he hecho han sido sobre esta rama. Pero si es un problema se puede quitar ya que ha cumplido su propósito

Reply:
Si vas a poner la rama principal y todas las ramas... Es mejor no poner ninguna. No sé por qué tenéis la manía de poner esto, no es obligatorio. Si vas a usar las imágenes Docker para testear, mejor no restringir la rama.